### ISSUE TITLE
Cavegen "griefs" old chunks that have been generated for some time

### ISSUE THREAD

**Issue by HeroOfTheWinds** [minetest/minetest#1559]
Apparently, in mgv6 (haven't tested in mgv7) the cavegen will make new caves in chunks that have been generated for a long time, and indiscriminately.  Inhabited areas complete with  cities can have a chunk of a stone building eaten out without warning, thus leading to the map essentially griefing itself.  Examples can be found on VanessaE's servers.

My proposal is this: have cavegen run once and only once, immediately after the chunk is generated.

**Comment by paramat** [comment]
Possibly related to #1397

**Comment by paramat** [comment]
^ @HeroOfTheWinds

**Comment by paramat** [comment]
Possibly related to #1397

**Comment by paramat** [comment]
Possible duplicate of #1397 which appears fixed.
Also no response so closing.
